test/gurka/gurka.h

```diff
@@ -640,13 +698,6 @@ valhalla::Api route(const map& map,
   return route(map, {from, to}, costing, options, reader);
 }
 
-valhalla::Api route(const map& map, const std::string& request_json) {
```

Comment:
i moved this one above and  had the other route method call it. i also wanted to delete the other route method but it was used by a lot of tests. there is a bit of redundancy here. i was trying to clean up a bit of it